cub/cub/agent/agent_select_if.cuh

```diff
@@ -764,6 +764,11 @@ struct AgentSelectIf
     InitializeSelections<true, IS_LAST_TILE>(
       tile_offset, num_tile_items, items, selection_flags, Int2Type<SELECT_METHOD>());
 
+    // Ensure that all items have been loaded before signalling subsequent CTAs that may potentially override this CTA's
+    // items in case of in-place compaction
+    __threadfence();
```

Comment:
Which strong load and stores pair with this fence?

Reply:
It's the loads on line 756/760 with the store happening within the call on line 783.

Reply:
Those loads are weak.

If the intent of this fence is to order these weak loads before the relaxed store on line 783, it suffices to make that one store a "release" store, e.g., by changing the `SetInclusive` API to support a `cuda::memory_order`, and then picking between relaxed and release for the store: https://github.com/NVIDIA/cccl/blob/259887a8023369cf2be7fee40113d1cc89df0e57/cub/cub/agent/single_pass_scan_operators.cuh#L612 .

Where is the atomic load that writes from the value that `SetInclusive`'s relaxed store writes to?